python/ray/ml/train/integrations/lightgbm/lightgbm_trainer.py

```diff
@@ -41,24 +48,78 @@ class LightGBMTrainer(Trainer):
             dataset. All datasets will be transformed by the ``preprocessor`` if
             one is provided. All non-training datasets will be used as separate
             validation sets, each reporting a separate metric.
-        lightgbm_config: LightGBM training parameters. Refer to
-            `LightGBM documentation <https://lightgbm.readthedocs.io/en\
-/latest/Parameters.html>`_ for a list of possible parameters.
+        lightgbm_config: LightGBM training parameters passed to ``lightgbm.train()``.
+            Refer to `LightGBM documentation <https://lightgbm.readthedocs.io>`_
+            for a list of possible parameters. Must contain a "params" key.
+            You can also pass a "dmatrix_params" key, which should be a dict
+            of ``dataset name:dict of kwargs`` passed to respective
+            :class:`xgboost_ray.RayDMatrix` initializations.
```

Comment:
should this be lightgbm ray?

```suggestion
            :class:`lightgbm_ray.RayDMatrix` initializations.
```

Reply:
It's the same object for both, even mentioned in LightGBM-Ray docs

Reply:
Hate to be the third person asking this, but if they're the same why not just use `lightgbm_ray` here?

Reply:
Because it will not link to docs correctly - we only have `xgboost_ray.RayDMatrix` in autodoc